ckan/logic/action/get.py

```diff
@@ -1254,8 +1254,9 @@ def _group_or_org_show(
         schema = group_plugin.db_to_form_schema()
 
     if include_followers:
+        context['session'] = model.Session
         group_dict['num_followers'] = logic.get_action('group_follower_count')(
-            {'model': model, 'session': model.Session},
+            context,
```

Comment:
If we pass None (or an empty dict) in for the context a new context will be generated with the default user determined from the request. This might be better than passing along `context` because often context contains internal parameters that only make sense for the original action it was passed to.
```suggestion
            None,
```

Reply:
This change don't work for my project.
I think this PR is clearer than just sending `None`, but I'm not sure if any specific item in "context" could be a problem.

Reply:
@avdata99 which fields do you actually need from context on your project? Was thinking of writing a `clean_context()` function that could copy just the minimum fields into a new context dict for cases like this (we have them everywhere in the code, and no consistent way of handling them)

Reply:
@avdata99 which fields do you actually need from context on your project? Was thinking of writing a `clean_context()` function that could copy just the minimum fields into a new context dict for cases like this (we have them everywhere in the code, and no consistent way of handling them)

Reply:
@wardi I updated this PR to include a `clean_context` function
Do you think is what we need?

Reply:
I think this is a great idea, but perhaps `clean_context()` implies emptiness (like our `clean_db`, `clean_index` test fixtures) and we actually mean "clean context but keeping the user and model". Maybe something like `fresh_context()`? Maybe I'm reading too much into it :)

Reply:
/polls clean_context fresh_context trim_context new_context minimal_context